src/System.Runtime.Environment/src/packages.config

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="System.Resources.ResourceManager" version="4.0.0-beta-22703" />
```

Comment:
I recommend rebasing this on master to pick up Eric's project.json change, moving these dependencies from packages.config to project.json, and bumping the version number of the packages.